Documentation/deploy-master.md

```diff
@@ -301,6 +287,69 @@ spec:
       timeoutSeconds: 1
 ```
 
+### Run the per host calico agent
+The per host Calico agent runs on all hosts including the master node. It is responsible for enforcing networking policy and connecting the pods on that host to the flannel overlay network.
+
+When creating `/etc/systemd/system/calico-node.service`:
+
+* Replace `${ETCD_ENDPOINTS}`
+* Replace `${ADVERTISE_IP}` with this node's publicly routable IP.
+
+**/etc/systemd/system/calico-node.service**
+
+```yaml
+[Unit]
+Description=Calico per-host agent
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+Slice=machine.slice
+Environment=CALICO_DISABLE_FILE_LOGGING=true
+Environment=HOSTNAME=${ADVERTISE_IP}
+Environment=IP=${ADVERTISE_IP}
+Environment=FELIX_FELIXHOSTNAME=${ADVERTISE_IP}
+Environment=CALICO_NETWORKING=false
+Environment=NO_DEFAULT_POOLS=true
+Environment=ETCD_ENDPOINTS=${ETCD_ENDPOINTS}
+ExecStart=/usr/bin/rkt run --inherit-env --volume=modules,kind=host,source=/lib/modules,readOnly=false --mount=volume=modules,target=/lib/modules --stage1-from-dir=stage1-fly.aci --insecure-options=image docker://quay.io/calico/node:v0.19.0
```

Comment:
Instead of using the docker:// prefix, we should be able to pull from quay and have it to convert to an ACI on the fly. Then the `--insecure-options=image` flag can be dropped too.